src/main/java/retriever/Ui.java

```diff
@@ -0,0 +1,205 @@
+package retriever;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import retriever.task.Task;
+
+/**
+ * The class handles the user interaction between the
+ * user and the chatbot.
+ */
+public class Ui {
+    private static String retrieverResponse;
```

Comment:
I like that you have this variable! Its a simple way to allow Retriever to work on both CLI and GUI 😄